apple/internal/codesigning_support.bzl

```diff
@@ -491,17 +491,17 @@ def _generate_codesigning_dossier_action(
 
     Args:
       actions: The actions provider from `ctx.actions`.
-      embedded_dossiers: An optional List of Structs generated from
-         `embedded_codesigning_dossier` that should also be included in this
-          dossier.
-      entitlements: Optional file representing the entitlements to sign with.
       label_name: Name of the target being built.
+      dossier_codesigningtool: The files_to_run for the code signing tool.
       output_discriminator: A string to differentiate between different target intermediate files
           or `None`.
       output_dossier: The `File` representing the output dossier file - the zipped dossier will be placed here.
+      embedded_dossiers: An optional List of Structs generated from
+         `embedded_codesigning_dossier` that should also be included in this
+          dossier.
+      entitlements: Optional file representing the entitlements to sign with.
```

Comment:
All three of these should be moved up to maintain alphabetical ordering.

Reply:
Done. (I also had to make the remaining arguments positional, but that apparently works.)